src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```
@@ -59,6 +59,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <GenerateRuntimeConfigurationFilesInputs Include="$(UserRuntimeConfig)" Condition=" Exists($(UserRuntimeConfig)) " />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworks)' != ''">
```

I think you also want to check that `TargetFramework` is set, so that errors generated in the outer build don't get `TargetFramework:` (with an empty value for the target framework) added.

```suggestion
  <ItemGroup Condition="'$(TargetFrameworks)' != '' And '$(TargetFramework)' != ''">
```